Kaffee.cs

```diff
@@ -22,9 +22,10 @@ public double wasserAuffuellen(double menge)
             return wasser;
         }
 
-        public double bohnenAuffuellen(double menge)
+        public double bohnenAuffuellen(double menge, Kaffeelager lager)
         {
             bohnen = ((bohnen + menge) > maxBohnen) ? maxBohnen : bohnen + menge;
+            lager.setLagerstand(lager.lagerstand + menge);
```

Comment:
Hier @martinernstmueller 
Es gibt 2 PRs von mir, einer von der Programmierübung (der hier) und der vom KaffeelagerController, den wir in der Stunde gemacht haben. Der vom KaffeelagerController haben wir ja vom main aus gemacht und da war die Lagerkorrektur noch nicht dabei und ich habe es vergessen die noch hinzufügen. Aber hier habe ich die Korrektur gemacht.